recipes/darker/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "darker" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3a840c2cedec7e4d13c26c450d94f31bd872375450a0c057b9276cc9c2dbad5c
+
+build:
+  # noarch: python
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - darker = darker.__main__:main
+
+requirements:
+  host:
+    - python  >=3.6
+    - pip
+  run:
+    - python  >=3.6
+    - black  >=20.0.0,<21.0.0
+    - toml
+    - typing-extensions  # [py<38]
+    - dataclasses        # [py<37]
+
+test:
+  imports:
+    - darker
+  requires:
+    - pip
+  commands:
+    - pip check
+    - test -f ${PREFIX}/bin/darker
```

Can you use something like this instead for testing?
```suggestion
    - darker --help
```